jekyll/_cci2/deployment_integrations.md

```diff
@@ -6,10 +6,24 @@ categories: [deploying]
 order: 1
 ---
 
+In the CircleCI 'Project Settings' page you will see a section called 'Continuous Deployment':
+
+![Deployment Integrations]({{ site.baseurl }}/assets/img/docs/deployment-integrations.png)
+
+Deployment options with built-in integration are listed there. This document exaplains how to use them with CircleCI 2.0.
+
 ## AWS
 
 ### Configuring AWS CLI
 
+To deploy to AWS from CircleCI 2.0 you will need to make sure the [awscli](http://docs.aws.amazon.com/cli/latest/userguide/installing.html) is installed in your primary container.
+
 You can add your AWS credentials from **Project Settings > AWS Permissions** page.
 
-The **Access Key ID** and **Secret Access Key** that you entered will be automatically available in your primary build container and exposed as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environmental variables.
+The **Access Key ID** and **Secret Access Key** that you entered will be automatically available in your primary build container and exposed as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables.
+
+## Heroku
+
+The built-in Heroku integration via the CircleCI UI is not yet implemented for CircleCI 2.0.
```

Comment:
It's not clear that we'll ever go back and do this the same way, so probably should change this language to be less of a promise for it to come back as-is.